src/shelly_component.hpp

```diff
@@ -36,6 +36,12 @@ class Component {
     kContactSensor = 9,
     kDoorbell = 10,
     kLightBulb = 11,
+    kSensor = 12,
+    kMax,
+  };
+
+  enum class SensorType {
```

Comment:
i'm not sure why SensorType is necessary. can you elaborate?

Reply:
it is not... yet. In the beginning I thought there were other W1 sensors also combined with Humidity.
AFAIK this would only be possible by an W1 to I2C bridge or by a non-onewire-sensor.

Reply:
understood. however, let's stick with official hardware so let's remove code related to supporting other sensor types for now.